### PR TITLE
Update Engineer role expectations

### PR DESCRIPTION
**Pull request by lukemorton**

### CODE REVIEW

File: roles/engineer.md [hunk 1]
```diff
@@ -1,118 +1,41 @@
 # Engineer
 
-Currently we view every role at Made Tech as an Engineer role. We expect this to change as we grow. Below you will see the variations of our engineer role along with the need for that role and our expectations for it.
+Our software delivery teams are the core of our business. They are usually comprised of a Lead Engineer, a Delivery Manager and one or more Engineers.
 
-## Engineer role
+As an Engineer we expect someone:
 
-We need someone:
+## To be able to autonomously deliver working software
```
Comment (chrisblackburn) [inline roles/engineer.md]: Prefer this without "To be able"

File: roles/engineer.md [hunk 2]
```diff
@@ -1,118 +1,41 @@
...
 
-We expect someone in the role:
+## To be fully billable
 
-- to be fully billable after 2 weeks on-boarding
-- to be involved in practice improvement discussions within 3 months
-- to be actively practicing modern software craftsmanship practices such as TDD within 3 months
-- to be autonomously delivering features of moderate complexity within 3 months
-- to be autonomously delivering features of high complexity within 6 months
+We expect our Engineers to be fully billed on customers deliveries as we are a business trying to make a profit by delivery valuable working software to our customers.
```
Comment (chrisblackburn) [inline roles/engineer.md]: delivering valuable

File: roles/engineer.md [hunk 3]
```diff
@@ -1,118 +1,41 @@
...
-We expect someone in the role:
+## To be able to productively pair with their colleagues
 
-- to be autonomously delivering features of a low complexity before graduation
-- to be actively practicing modern software craftsmanship practices such as TDD before graduation
-- to be involved in improving the academy program for current and future cohorts
-- to be billable at a 50% rate after graduating the Academy
+We expect our engineering team to be friendly and lovely people to pair with. All pairing dynamics are different, sometimes you will be pairing with someone with less experience, sometimes you will have the same experience. Adjusting the speed of your work to accomodate for your pair is important and special care must be taken when mentoring/coaching those with less experience. No matter who you are pairing with you must take turns to compromise when your opinions differ.
 
+## To be requesting peer reviews for the code they write
```
Comment (chrisblackburn) [inline roles/engineer.md]: / providing meaningful and considerate peer reviews of others code?

File: roles/engineer.md [hunk 4]
```diff
@@ -1,118 +1,41 @@
...
 
-## Engineer traits
+We expect our engineering team to always have their code peer reviewed at the least by one other person before merging. It's the responsibility of the code author to ask their colleagues to review and to merge the code in a timely fashion.
 
-### +Advocate
+## To regularly communicate with customers without reminder
 
-We need someone:
+We expect our Engineers to be communicating with our customers daily through their standups, but also throughout the day via slack. Everyone should feel empowered to ask customers questions, communicate progress and demonstrate successes.
```
Comment (chrisblackburn) [inline roles/engineer.md]: Slack

**Comment by chrisblackburn** [comment]
Something about attaining Core Competencies within a particular timeframe?
Something about delivering in accordance with Delivery Standards? (this may remove the need to be specific about things like TDD in this context)

**Comment by lukemorton** [comment]
I think we can add details about core competencies once they've been defined by learning. Same goes for delivery standards, once they're in the handbook we can add that expectation and link to it.

I'll put these both into my 90 day plan for People.

**Comment by lukemorton** [comment]
@chrisblackburn Have addressed your feedback.

**Pull request merged**